x/ecocredit/client/tx.go

```diff
@@ -49,17 +56,151 @@ func txCreateClass() *cobra.Command {
 			}
 			b, err := base64.StdEncoding.DecodeString(args[2])
 			if err != nil {
-				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "base64_metadata is malformed, proper base64 string is required")
+				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "metadata is malformed, proper base64 string is required")
 			}
 
-			c, ctx, err := mkMsgClient(cmd)
+			c, err := newMsgSrvClient(cmd)
 			if err != nil {
 				return err
 			}
-			_, err = c.CreateClass(cmd.Context(), &ecocredit.MsgCreateClassRequest{
+			msg := ecocredit.MsgCreateClassRequest{
 				Designer: args[0], Issuers: issuers, Metadata: b,
-			})
-			return mkTx(ctx, cmd, err)
+			}
+			_, err = c.client.CreateClass(cmd.Context(), &msg)
+			return c.send(err)
+		},
+	}
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+func txCreateBatch() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create_batch [issuer] [class_id] [metadata] [issuance]",
+		Short: "Issues a new credit batch",
+		Long: `Issues a new credit batch.
+
+Parameters:
+  issuer:    issuer address
+  class_id:  credit class
+  metadata:  base64 encoded issuance metadata
+  issuance:  JSON encode issuance list,
+             eg: '[{"recipient": "a1", "tradeableUnits": "10", "retiredUnits": "2"}]'`,
```

Comment:
```suggestion
  issuance:  JSON encode issuance list,
             eg: '[{"recipient": "a1", "tradableUnits": "10", "retiredUnits": "2"}]'
             Note: "tradableUnits" and "retiredUnits" default to 0.`,
```

Reply:
Add default note, and "tradeableUnits" should be "tradableUnits" (as is implemented in proto field name)